src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -421,10 +423,12 @@ private RootModule createRootModule() {
         final RootModule rootModule;
         try {
             final Properties props = createOverridingProperties();
-            final Configuration config =
-                ConfigurationLoader.loadConfiguration(
-                    configLocation,
-                    new PropertiesExpander(props),
+            final MultiThreadModeSettings singleThreadModeSettings =
+                    MultiThreadModeSettings.getSingleThreadMode();
```

Comment:
So ant will never be allowed to multi-thread? This doesn't seem right.
If we don't want to do this now, we need to have an issue to expand it later.